src/SceneKit/SCNCompat.cs

```diff
@@ -72,4 +76,28 @@ public unsafe virtual Task PresentSceneAsync (SCNScene scene, global::XamCore.Sp
 	}
 #endif
 #endif
+
+#if WATCH || XAMCORE_4_0
+	[Watch (4,0)]
+	[Mac (10,9), iOS (8,0)]
+	public delegate void SCNAnimationEventHandler (ISCNAnimationProtocol animation, NSObject animatedObject, bool playingBackward);
+#else
+	[Mac (10,9), iOS (8,0)]
+	public delegate void SCNAnimationEventHandler (CAAnimation animation, NSObject animatedObject, bool playingBackward);
+#endif
```

Comment:
`*Compat.cs` files are meant to be removed at some point. However the delegates will still be needed. Can't those just stay in the binding files ?

Reply:
Unfortunately I need to duplicate those in both the binding and the external file because if they are not used inside the binding file they are not generated at all inside `SupportDelegates.g.cs`

Reply:
good catch btw!